compiler/rustc_mir/src/borrow_check/diagnostics/conflict_errors.rs

```diff
@@ -1341,6 +1341,16 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
             suggestion,
             Applicability::MachineApplicable,
         );
+        if let Some(generator_kind) = use_span.generator_kind() {
+            if let GeneratorKind::Async(_) = generator_kind {
+                err.note(
+            "borrows cannot be held across a yield point, because the stack space of the current \
+            function is not preserved",
+        );
+                err.help("see https://rust-lang.github.io/async-book/03_async_await/01_chapter.html#awaiting-on-a-multithreaded-executor \
```

Comment:
You can add `// ignore-tidy-linelength` to the top of the file.

Reply:
When I add `// ignore-tidy-linelength` to the top of the file and then run `./x.py test tidy` I get the error `ignoring line length unnecessarily`.